libraries/AP_RangeFinder/AP_RangeFinder_TeraRangerI2C.cpp

```diff
@@ -13,43 +13,41 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 /*
-  driver for trone rangefinder
+  driver for TeraRanger I2C rangefinders
  */
-#include "AP_RangeFinder_trone.h"
+#include "AP_RangeFinder_TeraRangerI2C.h"
 
 #include <utility>
 #include <AP_HAL/AP_HAL.h>
 #include <AP_Math/crc.h>
 
 extern const AP_HAL::HAL& hal;
 
-#define TRONE_I2C_ADDR 0x30
-
 // registers
-#define TRONE_MEASURE 0x00
-#define TRONE_WHOAMI  0x01
-#define TRONE_WHOAMI_VALUE 0xA1
+#define TR_MEASURE 0x00
+#define TR_WHOAMI  0x01
+#define TR_WHOAMI_VALUE 0xA1
 
 /*
    The constructor also initializes the rangefinder. Note that this
    constructor is not called until detect() returns true, so we
    already know that we should setup the rangefinder
 */
-AP_RangeFinder_trone::AP_RangeFinder_trone(uint8_t bus, RangeFinder &_ranger, uint8_t instance, RangeFinder::RangeFinder_State &_state)
+AP_RangeFinder_TeraRangerI2C::AP_RangeFinder_TeraRangerI2C(uint8_t bus, RangeFinder &_ranger, uint8_t instance, RangeFinder::RangeFinder_State &_state)
     : AP_RangeFinder_Backend(_ranger, instance, _state, MAV_DISTANCE_SENSOR_LASER)
-    , dev(hal.i2c_mgr->get_device(bus, TRONE_I2C_ADDR))
+    , dev(hal.i2c_mgr->get_device(bus, _ranger._address[_state.instance]))
```

Comment:
I'd prefer this was handled the same way it is, for example, for LightWareI2C: check that address is set and pass it to the driver.